.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  minrust: 1.41.0
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        rust: [stable, beta, nightly]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: hecrj/setup-rust-action@v1
```

Comment:
Any reason not to use `actions-rs/..`?

Reply:
Sorry, for being :sloth: . I've updated the PR comment.